sklearn/ensemble/_weight_boosting.py

```diff
@@ -478,6 +478,10 @@ class AdaBoostClassifier(
     array([1])
     >>> clf.score(X, y)
     0.96...
+
+    For a detailed example of using AdaBoost to fit a sequence of DecisionTrees
+    as weaklearners, please refer to
+    :ref:`sphx_glr_auto_examples_ensemble_plot_adaboost_multiclass.py`.
```

Comment:
I think this is the only place where the mention makes sense. I would then remove the other two mentions in this PR.